### PR TITLE
HCAP-1376 - split documentation into multiple files

### PR DESCRIPTION
**Pull request by rhattie-ey**
Right now, the readme is very large, and difficult to navigate. This PR splits the readme into multiple files, and moves both these new files and the contents of the old /documents directory into /docs.

### CODE REVIEW

**Comment by fw-noel** [comment]
Hey @rhattie-ey , I wouldn't want to remove the README totally. Intro, ToC, Features seem like it would be good there

File: README.md [hunk 1]
```diff
@@ -86,143 +72,25 @@ Please note that this project is released with a [Contributor Code of Conduct](C
...
-
-- Run `make local-kc-build`
-- Run `make local-kc-run`
-- Run `make kc-import-users`
-Go to `http://keycloak.local.freshworks.club:8080/auth` console and add users/assign roles under the client `hcap-fe-local`
-- Run `make kc-export-users` to save your changes
-`409 Conflict` errors are ok if the user already exists, only the role change will be applied
 
-#### To run server unit tests:
+You should see the application running at `localhost:4000`.
```
Comment (fw-noel) [inline README.md:81]: Worth making this `http://hcapemployers.local.freshworks.club:4000` at this point?

Comment (rhattie-ey) [reply README.md:81]: Good point. I'll add it to the ticket I am writing for the new setup instructions (HCAP-1378).

**Review by vanics — APPROVED** [review]
I love the change. Often, we see the README growing uncontrollably on projects. This helps!

**Pull request merged**